.dependabot/config.yml

```diff
@@ -0,0 +1,27 @@
+version: 1
+update_configs:
+  - package_manager: ruby:bundler
+    directory: /
+    update_schedule: daily
+    allowed_updates:
+      # Security updates - we should always do these
+      - match: { update_type: security }
```

Comment:
Would this catch every security updates? This [PR](https://github.com/alphagov/frontend/pull/2364) for instance includes security updates but doesn't have the security label.

Reply:
That's interesting - good spot!

Fortunately, this would be covered by https://github.com/alphagov/content-publisher/pull/2062/files#diff-b44e3cec799fb1f7871015e85dba6eb3R25. Arguably this is a problem with Dependabot not marking the update correctly, which would already be a problem for any of our lower-level dependencies. For other gems, I guess we have to accept the risk.